src/FormatNumber.elm

```diff
@@ -113,13 +120,23 @@ format locale num =
                     -truncated
                         |> toFloat
                         |> format { locale | decimals = 0 }
-                        |> String.cons '−'
```

Comment:
If we are taking the sign into account separately, is it possible to avoid this `case … of` expression? Basically we can just go for `if abs truncated == 0 then "0" else …` where the `…` would be the same for positive and negative numbers. 

I'm saying that because all the `LT` part does is converting a negative `Int` number to a positive `Float` number and recurring to the same function (that is to say: what is really happening if the `GT` part, the `LT` can be replaced, with your PR, by `abs` and a `if` clause I guess).